agenda-webapps/src/main/webapp/vue-app/agenda/js/CalendarService.js

```diff
@@ -29,4 +29,41 @@ export function saveCalendar(calendar) {
       throw new Error('Error updating calendar');
     }
   });
+}
+
+export function saveAgendaSetting(contextKey, contextValue, scopeKey, scopeValue, key, value) {
```

Comment:
`saveAgendaSettings` instead to say that this is about multiple settings

Reply:
oaky